### PR TITLE
Fixes regression in python api CalcNumRotatableBonds

### PR DESCRIPTION
**Pull request by bp-kelley**
Re-adds CalcNumRotatableBonds(True) to python API

### CODE REVIEW

File: Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py [hunk 1]
```diff
@@ -350,7 +351,41 @@ def testSpiroAndBridgeheads(self):
...
-
-
+  def testNumRotatableBonds(self):
+    self.assertEquals(-1,
+           int(rdMD.NumRotatableBondsOptions.Default))
+    self.assertEquals(0,
+           int(rdMD.NumRotatableBondsOptions.NonStrict))
+    self.assertEquals(1,
+           int(rdMD.NumRotatableBondsOptions.Strict))
+    self.assertEquals(2,
```
Comment (greglandrum) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: It's not really a problem with the PR, but are the integer values of these constants actually important?

File: Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py [hunk 2]
```diff
@@ -350,7 +351,41 @@ def testSpiroAndBridgeheads(self):
...
+
+      v4 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.Default)
+      v5 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.NonStrict)
+      v6 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.Strict)
+      v7 = rdMD.CalcNumRotatableBonds(m, rdMD.NumRotatableBondsOptions.StrictLinkages)
+      
+      self.assertEquals(v1, v4)
+      self.assertEquals(v2, v5)
+      self.assertEquals(v3, v6)
+      print (v1, v2, v3, v4, v5, v6, v7)
```
Comment (greglandrum) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: I assume this is a debugging thing that doesn't need to be in the actual code?

File: Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py (hunk 1 above)
Comment (bp-kelley) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: They were when I was debugging, as it turns out the order that the variants of CalcNumRotatableBonds is defined in boost is actually important!

File: Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py (hunk 2 above)
Comment (bp-kelley) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: The print statement is, the rest are a regression test to assure that the True/False does the correct thing when compared to the enum.  I'm assuming you were referring to the print statement.

Comment (greglandrum) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: yeah, I was asking about the print statement.

File: Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py (hunk 1 above)
Comment (greglandrum) [inline Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py]: Weird. why is that?

**Pull request merged**